scripts/setupPGD.sh

```diff
@@ -0,0 +1,22 @@
+#################################################
+# Author: Chris Magnano
+#
+# 11/08/2019
+#
+# This script attempts to install the pgd library from github
+# in the lib directory. The lib directory is made if none exists.
+#################################################
+
+#Get repository directory and set up lib
+homeDir=`git rev-parse --show-toplevel`;
+mkdir ${homeDir}/lib;
+mkdir ${homeDir}/lib/pgd;
+
+#Get PGD from github and compile it
+git clone https://github.com/nkahmed/PGD.git ${homeDir}/lib/pgd/;
```

Comment:
Because there aren't official releases, we should clone from a specific commit.  Using the latest master could break our install process or parameter advising if the PDG code changes substantially.